src/molecules/formfields/ToggleField/index.js

```diff
@@ -64,10 +66,14 @@ function ToggleField( props ) {
 
   const message = meta && meta.touched && !meta.active && (meta.error || meta.warning);
 
+  const buttonStyle = sideBySide ? styles['side-by-side'] : styles['button-wrapper'];
+
   return (
     <div>
       <div className={classnames(...classes)}>
-        <Layout>
+        <Layout
+          nested={nested}
```

Comment:
Should this just always be nested? Also, why are there so many divs here?

Reply:
Idk. But, was afraid to remove anything for fear of maybe breaking other things? I can change it if need be??

Reply:
That's fair. I'd check with @jmcolella

Reply:
no it shouldn't always be nested. when we have the border surrounding the toggle buttons we need the outside padding. The extra div is what provides the border to the buttons and then the top level div wraps the entire toggle field and the associated error message.

Reply:
#boss man.

Reply:
Alright. Verdict is to leave it. Is this LGTM otherwise?